apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/IFlushableConsumer.cs

```diff
@@ -20,8 +23,15 @@ namespace Google.Cloud.Diagnostics.Common
     internal interface IFlushableConsumer<T> : IConsumer<T> 
     {
         /// <summary>
-        /// Flush the consumer.
+        /// Flushes the consumer.
+        /// </summary>
+        void Flushes();
```

Comment:
I think you may have misread my previous comment - the method should still be called `Flush`, but the *documentation* should describe what it does: It flushes the consumer.